src/main/java/brave/opentracing/BraveSpan.java

```diff
@@ -34,7 +45,111 @@ static BraveSpan wrap(brave.Span span) {
     return new BraveSpan(span);
   }
 
-  private BraveSpan(brave.Span delegate) {
-    super(delegate);
+  /**
+   * Converts an existing {@linkplain Span} for use in OpenTracing apis
+   */
+  public final brave.Span unwrap() {
```

Comment:
I assume code below here is copy/paste from prior. PS the javadoc here is backwards.